hydra_server/server/complexmodels.py

```diff
@@ -883,7 +883,7 @@ class ResourceSummary(HydraComplexModel):
         ('types',       SpyneArray(TypeSummary)),
     ]
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, include_attributes=True):
         super(ResourceSummary, self).__init__()
 
         if parent is None:
```

Comment:
The intention here isn't clear to me; perhaps an explanatory comment if this is correct?

Should 890 be `return` instead of `parent` as seems to allow `get_attrs` in 891-893 on a `None`?